src/components/Modal/AddModal.tsx

```diff
@@ -0,0 +1,88 @@
+import axios from 'axios';
+import React from 'react';
+import Button from 'react-bootstrap/Button';
+import Modal from 'react-bootstrap/Modal';
+import { useCookies } from 'react-cookie';
+import { useForm } from "react-hook-form";
+
+type handleShowType = {
+  showAdd: boolean;
+  handleCloseAdd: () => void;
+}
+
+function AddModal(props: handleShowType) {
+  const { register, handleSubmit } = useForm();
+  const {showAdd, handleCloseAdd} = props
+  const [ cookies ] = useCookies(["token"]);
+
+  let token = cookies.token;
+  let config = {
+    headers: {
+      Authorization: "Bearer " + token,
+    }
+  }
+
+  const handleAdd = (data: any) => {
+    axios.post('http://localhost:4000/barang', data, config).then(response => {
+      console.log("ini nilai respon", response )
```

Comment:
remove log

Reply:
Done delete all log, commets and unused imports